### PR TITLE
Simplify installation and config instructions

### PR DESCRIPTION
**Pull request by teohhanhui**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ In config.js:
 ``` javascript
 System.config({
   map: {
-    "googlmaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!HeinrichFilter/systemjs-plugin-googlemaps/googlemaps"
+    "googlemaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!systemjs-googlemaps"
```
Comment (HeinrichFilter) [inline README.md]: Thanks for the PR! I like the idea of simplifying the install/config.

But does this work for you as it is? I had to change mine to:

```
"googlemaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!systemjs-googlemaps/googlemaps"
```

to get it to work.

Did you change anything else in your config.js?

Comment (teohhanhui) [inline README.md]: Actually I haven't tried it :stuck_out_tongue:

I think we should fix the loading of main anyway... Will look into it later.

Comment (HeinrichFilter) [inline README.md]: Thanks!

The `/googlemaps` at the end was annoying me but I couldn't quite figure out how to get rid of it. If you could help me to fix/load main, that would be great. (I'm quite new to SystemJS/JSPM)

Do you know where it is documented?

**Comment by teohhanhui** [comment]
Switched to `meta.<pkg>.loader` config since syntax-based plugin loading in `map` config is no longer supported according to systemjs/systemjs#481

> Unfortunately mapping into plugins is something that is no longer supported in the latest release as it results in circular normalization.

**Comment by HeinrichFilter** [comment]
Thanks @teohhanhui, but it seems like I may have merged too early

This fails on my machine. Was it working for you?

I'll revert the changes if we can't figure it out quickly :-/

**Comment by teohhanhui** [comment]
Yes. Works for me. What's the issue you're facing?

**Comment by HeinrichFilter** [comment]
My bad, sorry, ignore that. It is working :)

I've tried so many different things during the day. My project was just in a weird state.

Do you know whether this means that all the plugins listed at the top of the [JSPM registry](https://github.com/jspm/registry/blob/master/registry.json) also can't be used with `!plugin` anymore?

Either way, thanks for your help!

**Comment by teohhanhui** [comment]
IIRC it doesn't work when used in `map` (I tried doing some overrides in there last time for some json files. Now I understand why it didn't work...)

**Comment by teohhanhui** [comment]
It should work just fine everywhere else.

**Comment by HeinrichFilter** [comment]
Aaah, I think I see what you mean. So the following works:
`System.import('https://maps.googleapis.com/maps/api/js?libraries=places!systemjs-googlemaps').then(function (GoogleMaps) {...})`

But you can't use that string in `map` since that then causes the circular normalization (as per systemjs/systemjs#481)?

**Pull request merged**